package.json

```diff
@@ -114,7 +114,7 @@
   "bundlesize": [
     {
       "path": "./dist/reflex.min.js",
-      "maxSize": "8 Kb"
+      "maxSize": "9.5 Kb"
```

Comment:
This feature is fairly large as it is now (1.5kb more), I'm open to removing features if we think it's worth it